packages/logger/package.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "@sap-ux/logger",
+    "version": "0.1.0",
+    "description": "A simple logging module",
+    "license": "SEE LICENSE IN LICENSE",
+    "private": false,
+    "main": "dist/index.js",
+    "scripts": {
+        "build": "pnpm clean && tsc",
+        "watch": "tsc --watch",
+        "clean": "rimraf dist coverage",
+        "format": "prettier --write '**/*.{js,json,ts,yaml,yml}' --ignore-path ../../.prettierignore",
+        "lint": "eslint . --ext .ts",
+        "lint:fix": "eslint . --ext .ts --fix",
+        "test": "jest --maxWorkers=1 --colors",
+        "test-u": "jest --maxWorkers=1 --colors -u",
```

Comment:
I recall you asked to remove `test-u` from the other modules

Reply:
I kept this to be consistent with the other modules. Shall I remove this from all the modules?

Reply:
may as well

Reply:
Changed my mind, don't want to make changes in the other modules here. Removed in logger, ta.